.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+go_import_path: github.com/src-d/metdata-retrieval
+go:
+  - 1.13.x
+env:
+  global:
+    - SOURCED_GITHUB_TOKEN=$GITHUB_TOKEN
```

Comment:
Why're using `GITHUB_TOKEN` for the env var storing the github token
https://github.com/src-d/metadata-retrieval/pull/22/files#diff-fc4f49f574c8bbba1fbc6c64589a5536R49